### PR TITLE
chore(ldoc: css): new style for the tables

### PR DESCRIPTION
**Pull request by actionless**
2018-03-20--1521564782_3357x1004_scrot

### CODE REVIEW

**Comment by actionless** [comment]
i found problem with function_list and module_list styles overlapping, hope now it's fine

File: docs/ldoc.css [hunk 1]
```diff
@@ -143,12 +143,15 @@ pre {
...
-#navigation ul
-{
+table.index td {
+    text-align: left;
+    vertical-align: top;
+}
+
+#navigation ul {
     font-size:1em;
     list-style-type: none;
```
Comment (Elv13) [inline docs/ldoc.css]: Can I edit the PR to remove that line or can you add a fixup for it?

Comment (actionless) [reply docs/ldoc.css]: i think the one which you wanted is on line 266

**Comment by actionless** [comment]
@Elv13 

2018-03-20--1521567535_3359x2103_scrot

**Comment by Elv13** [comment]
:+1:

**Comment by actionless** [comment]
oops, when rebasing from github UI it's not sticking !fixup commits to their parents :(

**Comment by Elv13** [comment]
well, too late for that

**Comment by Elv13** [comment]
and thanks for this PR, looks much better now!

**Pull request merged**